packages/eslint-plugin/package.json

```diff
@@ -38,19 +39,24 @@
     "accessibility"
   ],
   "dependencies": {
-    "@eslint/plugin-kit": "0.2.8",
+    "@eslint/plugin-kit": "0.3.1",
     "@html-eslint/parser": "^0.40.0",
     "@html-eslint/template-parser": "^0.40.0",
     "@html-eslint/template-syntax-parser": "^0.40.0"
   },
+  "peerDependencies": {
+    "eslint": "^9.7.0"
```

Comment:
Replacement for the types, also sometimes needed for strict package managers (`hoist: false` stuff).

Reply:
@lishaduck There are many users still using v8 and this plugin supports v8. What do you think about using “>=8”?

Reply:
Oh, I assumed that because it supported languages it didn't still support v8. Will revert this + associated cleanups.